composables/timeline.ts

```diff
@@ -11,7 +11,7 @@ function areStatusesConsecutive(a: mastodon.v1.Status, b: mastodon.v1.Status) {
 
 function removeFilteredItems(items: mastodon.v1.Status[], context: mastodon.v1.FilterContext): mastodon.v1.Status[] {
   const isStrict = (filter: mastodon.v1.FilterResult) => filter.filter.filterAction === 'hide' && filter.filter.context.includes(context)
-  const isFiltered = (item: mastodon.v1.Status) => !item.filtered?.find(isStrict)
+  const isFiltered = (item: mastodon.v1.Status) => item.account.id === currentUser.value?.account.id || !item.filtered?.find(isStrict)
```

Comment:
@danielroe I found that `currentUser` here is causing the unit test failure. Does it need the window object somehow?

Reply:
moved to happy-dom